tests/playbooks/tests_bond_cloned_mac.yml

```diff
@@ -9,7 +9,25 @@
     dhcp_interface1: test1
     port2_profile: bond0.1
     dhcp_interface2: test2
+    ansible_ssh_common_args: '-o ControlMaster=auto -o ControlPersist=no'
```

Comment:
?

Reply:
I saw the ssh failure on rhel-6 testing: 
```
ansible failure: Make sure this host can be reached over ssh: mux_client_request_session: read from master failed: Broken pipe
```
The above `ansible_ssh_common_args` seems to fix the mux_client_request_session failure.